### ISSUE TITLE
duplicate titles when embedding an entire notebook in a wordpress post

### ISSUE THREAD

**Issue by arkraieski** [andybrowncompassion/observable-embed#1]
A problem I've been having with various methods of embedding observable notebooks in WP (this plugin, iframe, etc.) is that embedding an entire notebook results in having 2 titles appear: one from giving the post a title in WordPress and the other from giving the notebook a title on obserablehq. I'll attach a screenshot of an example of what I mean at the bottom, but I think you get the idea.

I have a "fix" for this running in production on my personal website; a simple addition to public/css/observable-embed-public.css:

```
div.observablehq h1 {
    display: none;
}
```

I was thinking that it might make sense to have an options page in admin for toggling frontend tweaks like this? Not sure what would be involved there

**Comment by andybrowncompassion** [comment]
Thanks, @arkraieski . I think an option to hide, or better - ignore that cell, would be ideal.

**Comment by arkraieski** [comment]
@andybrowncompassion Yeah, I think I agree. I'm not sure how to implement "ignoring" though. The css tweak above that I'm using is "hiding" imo, but without an option. 

Also, thinking more generally/long-term, I'm sure myself and others will come up with other little CSS additions we like to use with this plugin (since there are so many different ways to use Observable and WordPress), so some sort of general way to turn things like this into options could make it easier for people to contribute.
